c7n/filters/health.py

```diff
@@ -29,12 +29,20 @@ class HealthEventFilter(Filter):
         types={'type': 'array', 'items': {'type': 'string'}},
         statuses={'type': 'array', 'items': {
             'type': 'string',
-            'enum': ['open', 'upcoming']
+            'enum': ['open', 'upcoming','closed']
         }})
 
     permissions = ('health:DescribeEvents', 'health:DescribeAffectedEntities',
                    'health:DescribeEventDetails')
 
+    def validate(self):
+        if self.data.get('types'):
```

Comment:
this seems suspect. what are the values for types in the config here? this could use a doc string. its also unclear what the purpose is. effectively it seems to be saying .. resource_type in types but types here corresponds to event types.